packages/mui-base/src/useSelect/useSelect.ts

```diff
@@ -27,6 +27,44 @@ import { selectReducer } from './selectReducer';
 import { combineHooksSlotProps } from '../utils/combineHooksSlotProps';
 import { MuiCancellableEvent } from '../utils/MuiCancellableEvent';
 
+// visually hidden style based on https://webaim.org/techniques/css/invisiblecontent/
+const visuallyHiddenStyle: React.CSSProperties = {
+  clip: 'rect(1px, 1px, 1px, 1px)',
+  clipPath: 'inset(50%)',
+  height: '1px',
+  width: '1px',
+  margin: '-1px',
+  overflow: 'hidden',
+  padding: 0,
+  position: 'absolute',
+  left: '50%',
+  bottom: 0, // to display the native browser validation error at the bottom of the Select.
+};
+
+const noop = () => {};
+
+function defaultFormValueProvider<OptionValue>(
```

Comment:
Moved from Base UI Select

Reply:
We can remove the one from the Select now, right?